deploy-osp-adoption.yml

```diff
@@ -0,0 +1,108 @@
+---
+# Purpose of this playbook:
+# This playbook deploy OSP 17.1 on previously created
+# infra for adoption. The main goal of this playbook
+# is to be called as a standalone piece of a bigger
+# job, mostly in the adoption context.
+#
+# You would typically run this playbook after the
+# "create-infra.yml"
+
+- name: Deploy 17.1 for adoption
+  hosts: 'localhost'
```

Comment:
this one is a bit tricky, I need to use the `cifmw_target_host` var because some tasks need to be delegated to the hypervisor (like starting vms or fetching the network env defintion) but the playbook itself needs to run from localhost otherwise it does not find the hook to deploy ceph (I don't have ci-framework on the hypervisor)

Reply:
ack, understood mate.